src/main/java/seedu/duke/lesson/Lesson.java

```diff
@@ -1,14 +1,35 @@
 package seedu.duke.lesson;
 
+import seedu.duke.ui.UI;
+
+import java.util.ArrayList;
+
 public class Lesson {
 
+    public static final String DELIMITER = ";;";
+    public static final int NUMBER_OF_ARGUMENTS = 5;
+    public static final String TUTORIAL = "TUTORIAL";
+    public static final String LECTURE = "LECTURE";
+    public static final String LAB = "LAB";
+    public static final int LESSON_TYPE_INDEX = 0;
+    public static final int TIME_INDEX = 1;
+    public static final int ONLINE_LINK_INDEX = 2;
+    public static final int TEACHING_STAFF_NAME_INDEX = 3;
+    public static final int TEACHING_STAFF_EMAIL_INDEX = 4;
```

Comment:
These can be shifted to a Constants class in the common package when we merge.

Reply:
Was planning on doing so after I'm done implementing all commands. Thank you for the suggestion anw.